src/modules/cvrp/processor/vrp_processor.py

```diff
@@ -387,11 +404,11 @@ def map_slo_cro_vehicles(self, vehicles, event_type):
 
         # First graph is SLO, second graph is CRO
         for v in vehicles:
-            if "S" in v.start_node or "PS" in v.start_node:
+            if "SLO" == v.country:
```

Comment:
maybe add this in constants if you have time..

Reply:
fixed